tests/test_portchannel_global.rb

```diff
@@ -215,4 +215,26 @@ def test_load_balance_asym_rot
     assert_equal(global.default_asymmetric, global.asymmetric)
     assert_equal(global.default_rotate, global.rotate)
   end
+
+  def test_load_balance_no_hash_rot
+    global = create_portchannel_global
+    unless validate_property_excluded?('portchannel_global', 'hash_poly')
```

Comment:
Let us discuss this. The code is correct even though it looks odd, I had to do this way as I could not find a cleaner approach